app/jobs/bulk_actions_job.rb

```diff
@@ -36,7 +36,7 @@ def bulk_remove_labels
   def available_params(params)
     return unless params[:fields]
 
-    params[:fields].delete_if { |_k, v| v.nil? }
+    params[:fields].delete_if { |k, _v| k.nil? }
```

Comment:
@tejaswinichile, can the keys here be nil ? Do we need this? 
Also, should we validate the keys in params[:fields]

Reply:
@sojan-official Yes, this is for bulk action; if keys are present, then only consider updating the values.
The last changes with values nil not helping to update assignee_id to empty/nil.
We are validating these keys in the bulk action controller.